clusterctl/examples/google/cluster.yaml.template

```diff
@@ -0,0 +1,19 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: Cluster
+metadata:
+  name: $CLUSTER_NAME
+  annotations:
+    gce.clusterapi.k8s.io/service-account-k8s-master: "master-$CLUSTER_NAME@$GCLOUD_PROJECT.iam.gserviceaccount.com"
```

Comment:
can you add a comment of how this annotation is used

annotations are free formed, so any documentation about how this will be used will be helpful

Reply:
Added some comments.